wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -711,6 +719,26 @@ extension ZMUserSession: ZMNetworkStateDelegate {
         networkState = state
     }
 
+    public func getE2eIdentityCertificates(for clients: [MLSClientID]) async throws -> [E2eIdentityCertificate] {
+        guard let conversationId = await fetchSelfConversation(context: syncContext) else {
+            return Array(repeating: .notActivated, count: clients.count)
+        }
+        return try await getE2eIdentityCertificatesUseCase.invoke(
+            conversationId: conversationId,
+            coreCryptoProvider: coreCryptoProvider,
+            clientIds: clients
+        )
+    }
+
+    public func getIsE2eIdentityEnabled() async throws -> Bool {
+        return try await getIsE2eIdentityEnabledUseCase.invoke(coreCryptoProvider: coreCryptoProvider)
+    }
```

Comment:
One of the benefits of the use case pattern is that you can inject them where you need them in the UI. If we wrap them in functions like this the UI code depends on the  whole `ZMUserSession` which makes things hard to test.

Reply:
done